src/models/slices/query.ts

```diff
@@ -1,10 +1,32 @@
 import { QuerySource, QueryTrigger, SearchIntent } from '@yext/answers-core';
 
+/**
+ * Maintains the latest query and its associated data.
+ */
 export interface QueryState {
+  /**
+   * The latest user query input.
```

Comment:
What do you think about wording this as "The input used in the next search"

Reply:
I think it would be good to still specify that it's user input. Maybe "The user input for the next query to search"?

Reply:
Maybe "The user input used for the next search query"?

Reply:
Yes, that works!